optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/domain/ConferenceParametrization.java

```diff
@@ -34,6 +34,8 @@
     public static final String SPEAKER_PROHIBITED_ROOM_TAG = "Speaker prohibited room tag";
     public static final String TALK_REQUIRED_ROOM_TAG = "Talk required room tag";
     public static final String TALK_PROHIBITED_ROOM_TAG = "Talk prohibited room tag";
+    public static final String TALK_MUTUALLY_EXCLUSIVE_TALKS_TAG = "Talk mutually-exclusive-talks tag";
```

Comment:
@ge0ffrey The reason behind the name is that "mutually-exclusive" is a property for plurals, i.e. there's no one mutually-exclusive talk, it has to be "many" talks that are mutually-exclusive, however we can have on prohibited/preferred room.
I would've named it TALK_MUTUALLY_EXCLUSIVE_TALK_TAG"S" but kept it as "TAG" because all other fields are named the same.

Reply:
your reasoning makes sense.
The bug is that the constraint "Speaker required room tag" check all room tags of a talk at once (it uses missingSpeakerRequiredRoomTagCount in the DRL), so it should be named plural, because that how we want it to show in the justification/indictment list (the last yellow sheet in the spreadsheet "score explanation").

So we you mind renaming the score rules, as well as these constants to a plural if and only if the score rule DRL checks all tags of one talk at once?

Reply:
@MusaTalluzi I am going to merge this PR already, would you mind handling those plurals in a separate PR?